src/Microsoft.ML.Data/Evaluators/Metrics/BinaryClassificationMetrics.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Linq;
 using Microsoft.ML.Runtime;
```

Comment:
If I look through this file, it's unclear to me which of the changes require this.